packages/c/catch2/xmake.lua

```diff
@@ -29,11 +30,23 @@ package("catch2")
     end
 
     on_load(function (package)
+       if package:config("use_catch2main") then
+            package:add("components", "main")
+        end
+        package:add("components", "lib")
+    end)
+
+    on_component("main", function (package, component)
+        if package:version():ge("3.0") then
+            component:add("links", "Catch2Main")
+        end
+    end)
+
+    on_component("lib", function (package, component)
         if package:version():ge("3.0") then
-            package:add("deps", "cmake")
```

Comment:
why.remove it?

Reply:
you can add package deps in on_load

Reply:
Sorry, that is fixed in the latest commit